test/test-app/package.json

```diff
@@ -7,7 +7,7 @@
   "scripts": {
     "dev": "nodemon --ignore node_modules/ server.js",
     "postinstall": "tsc greeting.ts && node-gyp -C addon configure build",
-    "start": "echo 'starting node app' && node server.js"
+    "start": "node server.js"
```

Comment:
So, if the start script has multiple chained commands, this can be a problem for the docker exit code? If so, I think we definitely need to document this.

Reply:
Yeah, I need to look into that more closely. I spent way too much time yesterday trying to cover that as well (https://github.com/danbev/centos7-s2i-nodejs/commit/a3c5c550331bac4508e5cf3f0d190504f294f977) but I'm not sure if it is stable enough and worth it.